recipe/migrations/gstreamer121.yaml

```diff
@@ -1,7 +1,9 @@
 __migrator:
   build_number: 1
   kind: version
-  migration_number: 1
+  migration_number: 2
 gstreamer:
 - '1.21'
-migrator_ts: 1664900131.838455
+gst_plugins_base:
+- '1.21'
+migrator_ts: 1667172469
```

Comment:
Is this the case for all PRs? Otherwise, we don't need to restart the migration and a rerender on the respective PRs should be enough.

Reply:
The PRs that went through should be ok. I'll undo the changes to the timestamp and migrator number.